app/src/Containers/ProjectSearchResultsContainer.tsx

```diff
@@ -27,15 +27,14 @@ const ProjectSearchResultsContainer = (props: {
     }
   }
 }) => {
-  const { results, resultsType, searchField, searchQuery } = props.route.params
+  const { results, searchField, searchQuery } = props.route.params
 
   if (results) {
     return (
       <SafeArea>
         <TopOfApp />
         <SearchTerm>
-          Results for {searchField ?? ''}
```

Comment:
Should this actually say 'cause' instead of 'charity sector' if the field is `sector`?  That way it would be consistent with the search screen, which might make more sense to the user?

Reply:
Agree that would be better yes, will update. Thanks, Joe

Reply:
That's it done, thanks for improving Joe :)